crates/sui-graphql-e2e-tests/tests/transactions/errors.exp

```diff
@@ -20,7 +20,7 @@ Response: {
         {
           "effects": {
             "status": "FAILURE",
-            "errors": "Move Runtime Abort. Location: 83f3be7571dccac8f0fd562cf4aa66707bafe3996c49b6f659be49a9841653fe::m::boom (function index 1) at offset 1, Abort Code: 42 in 1st command."
+            "errors": "Error in 1st command, Move Runtime Abort. Location: 83f3be7571dccac8f0fd562cf4aa66707bafe3996c49b6f659be49a9841653fe::m::boom (function index 1) at offset 1, Abort Code: 42"
```

Comment:
@amnn I believe this change is fine since it's just changing the format of a string value, but want to double-check before landing that this change to existing (non-clever) Move aborts is OK?

Reply:
Yeah it's all good (but this is a good call out on what I meant by my earlier question about error formats -- it's this that I wanted to align with the clever error format, or vice versa).